_includes/selector.html

```diff
@@ -728,6 +640,11 @@
                 if (this.active_release === "Nightly" && method === "pip") isDisabled = true;
                 return isDisabled;
             },
+            disableUnsupportedPackage(package) {
+                var isDisabled = false;
+                if (this.active_cuda_ver.startsWith("12") && this.active_method === "Conda" && package === "cuSignal") isDisabled = true;
```

Comment:
Do we need to disable cuSignal for 23.10 nightlies? See https://docs.rapids.ai/notices/rsn0032/.
```suggestion
                if (package === "cuSignal" && ((this.active_cuda_ver.startsWith("12") && this.active_method === "Conda") || this.active_release === "Nightly")) isDisabled = true;
```